includes/entities/class-fs-plugin.php

```diff
@@ -59,6 +59,17 @@ class FS_Plugin extends FS_Scope_Entity {
 		 * @var bool
 		 */
 		public $is_live;
+		/**
+         * `false` if the module doesn't have an affiliate program or one of the following: `selected`, `customers`,
```

Comment:
@fajardoleo the proper place to put the description of the property is after its type (i.e. after `@var string|bool`). I also think that you can specify `string|false` as the type to be more accurate (make sure the IDE doesn't show a warning for it).